### PR TITLE
Add runtime configuration

### PR DESCRIPTION
**Pull request by gfphoenix78**
## Description

coordinator folder is removed, now all source code of plcontainer and coordinator is placed in src. It is too hard to separate code of coordinator from plcontainer, so we simply compile more source code than needed.

Note: Runtime configuration loading and reloading are both done in coordinator process.

Co-authored-by: Xinyue Ge <xge@pivotal.io>
Co-authored-by: Hao Wu <hawu@pivotal.io>
## Tests

coordinator background worker runs well, no test currently.

### CODE REVIEW

File: src/Makefile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        common/base_network.c common/comm_channel.c common/comm_connectivity.c \
+        common/comm_dummy_plc.c common/comm_messages.c
+OBJS = $(foreach src,$(FILES),$(subst .c,.o,$(src)))
+
+override SHLIB_LINK += $(shell pkg-config --libs json-c)
+override SHLIB_LINK += $(shell xml2-config --libs)
+override SHLIB_LINK += $(shell curl-config --libs)
+
+INCLUDE_DIR = include
+override CFLAGS += -std=gnu99 -I$(INCLUDE_DIR) -Wall -Wextra
```
Comment (HaozhouWang) [inline src/Makefile]: no need to add -std=gnu99, GPDB already does this

File: src/plc_coordinator.c [hunk 2]
```diff
@@ -118,6 +117,15 @@ plc_initialize_coordinator()
 
     sock = plc_listen_socket();
 
+    if (plc_refresh_container_config(false) != 0) {
+        if (runtime_conf_table == NULL) {
+            /* can't load runtime configuration */
+            elog(ERROR, "PL/container: can't load runtime configuration");
```
Comment (HaozhouWang) [inline src/plc_coordinator.c]: chang to plc_elog?

Comment (gfphoenix78) [reply src/plc_coordinator.c]: > chang to plc_elog?

Forgot it. This is the old code before plc_elog refactored.

File: src/Makefile (hunk 1 above)
Comment (gfphoenix78) [reply src/Makefile]: removed

**Pull request merged**